<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../qbittorrent.png</normaloff>../qbittorrent.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_filter">
       <attribute name="title">
        <string>Filters</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Active</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Paused</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Completed</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeView" name="filters_Browser"/>
           <widget class="QTreeView" name="torrents_Browser"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_search">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="search_bar">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_searchcateg"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_search">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_add_search_as_filter">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Add Search as Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="status_label">
            <property name="text">
             <string>Status: Not Running</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="resultsBrowser"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_download">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_description">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Go to Description Page</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="separator"/>
   <addaction name="actionOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>Create Filter</string>
   </property>
   <property name="iconText">
    <string>Create Filter</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="iconText">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>&amp;Resume</string>
   </property>
   <property name="iconText">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="iconText">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="iconText">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/skin/qbittorrent32.png</normaloff>:/icons/skin/qbittorrent32.png</iconset>
   </property>
   <property name="text">
    <string>Visit &amp;Website</string>
   </property>
  </action>
  <action name="actionDownload_from_URL">
   <property name="text">
    <string>Add &amp;link to torrent...</string>
   </property>
   <property name="iconText">
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionCreate_torrent">
   <property name="text">
    <string>Torrent &amp;creator</string>
   </property>
  </action>
  <action name="actionBugReport">
   <property name="text">
    <string>Report a &amp;bug</string>
   </property>
  </action>
  <action name="actionSet_upload_limit">
   <property name="text">
    <string>Set upload limit...</string>
   </property>
  </action>
  <action name="actionSet_download_limit">
   <property name="text">
    <string>Set download limit...</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
  </action>
  <action name="actionSet_global_download_limit">
   <property name="text">
    <string>Set global download limit...</string>
   </property>
  </action>
  <action name="actionSet_global_upload_limit">
   <property name="text">
    <string>Set global upload limit...</string>
   </property>
  </action>
  <action name="actionBottomPriority">
   <property name="text">
    <string>Minimum priority</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTopPriority">
   <property name="text">
    <string>Top priority</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDecreasePriority">
   <property name="text">
    <string>Decrease priority</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionIncreasePriority">
   <property name="text">
    <string>Increase priority</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUse_alternative_speed_limits">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alternative speed limits</string>
   </property>
   <property name="toolTip">
    <string>Alternative speed limits</string>
   </property>
  </action>
  <action name="actionTop_tool_bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top &amp;tool bar</string>
   </property>
   <property name="toolTip">
    <string>Display top tool bar</string>
   </property>
  </action>
  <action name="actionSpeed_in_title_bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Speed in title bar</string>
   </property>
   <property name="toolTip">
    <string>Show transfer speed in title bar</string>
   </property>
  </action>
  <action name="actionRSS_Reader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;RSS reader</string>
   </property>
  </action>
  <action name="actionSearch_engine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search &amp;engine</string>
   </property>
  </action>
  <action name="actionLock_qBittorrent">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock qBittorrent</string>
   </property>
   <property name="iconText">
    <string>Lock</string>
   </property>
   <property name="toolTip">
    <string>Lock qBittorrent</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+L</string>
   </property>
  </action>
  <action name="action_Import_Torrent">
   <property name="text">
    <string>Import existing torrent...</string>
   </property>
   <property name="toolTip">
    <string>Import torrent...</string>
   </property>
  </action>
  <action name="actionDonate_money">
   <property name="text">
    <string>Donate money</string>
   </property>
   <property name="toolTip">
    <string>If you like qBittorrent, please donate!</string>
   </property>
  </action>
  <action name="actionStart_All">
   <property name="text">
    <string>R&amp;esume All</string>
   </property>
  </action>
  <action name="actionPause_All">
   <property name="text">
    <string>P&amp;ause All</string>
   </property>
  </action>
  <action name="actionExecution_Logs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Execution &amp;Log</string>
   </property>
   <property name="toolTip">
    <string>Execution Log</string>
   </property>
  </action>
  <action name="actionAutoExit_qBittorrent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit qBittorrent</string>
   </property>
  </action>
  <action name="actionAutoSuspend_system">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Suspend system</string>
   </property>
  </action>
  <action name="actionAutoHibernate_system">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hibernate system</string>
   </property>
  </action>
  <action name="actionAutoShutdown_system">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shutdown system</string>
   </property>
  </action>
  <action name="actionAutoShutdown_Disabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
  </action>
  <action name="actionToggleVisibility">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string notr="true">Minimize</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="toolTip">
    <string>Check for program updates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
